src/main/java/org/geysermc/cumulus/form/SimpleForm.java

```diff
@@ -124,6 +133,17 @@ interface Builder extends FormBuilder<Builder, SimpleForm, SimpleFormResponse> {
     @This
     Builder button(@NonNull String text);
 
+    /**
+     * Adds a button directly to the Form, but only when shouldAdd is true.
+     *
+     * @param button    the button to add
+     * @param shouldAdd if the button should be added
+     * @return the form builder
+     * @since 1.1
+     */
+    @This
+    Builder optionalButton(@NonNull ButtonComponent button, boolean shouldAdd);
```

Comment:
I'm not sure what the use of this method would be. If a button shouldn't be added then I'm not sure why the instance was created in the first place.

Reply:
I don't have use for directly adding buttons anymore, but if I did, it is true I would instead check if it should be shown, and then if not, just add a blank component instead of doing some placeholder work (in order to get button IDs to lineup). I'll remove this